bi_view_editor/__manifest__.py

```diff
@@ -11,7 +11,9 @@
     "category": "Productivity",
     "version": "16.0.1.0.0",
     "development_status": "Beta",
-    "depends": ["web"],
+    "depends": [
+        "spreadsheet_dashboard",
```

Comment:
⚠️ do we really want to add the `spreadsheet` dependency just for this menu?

Reply:
i also think that this is a quite big and useless dependency. what do you suggest? put it in `base.menu_board_root` by default and add a glue module (auto-install) that depend on `spreadsheet_dashboard` and moves the menu entry there?

what do you think, @PierrickBrun?

Reply:
That sounds good to me 👍🏻